public/html/reports/EditReportViewModel.html

```diff
@@ -0,0 +1,110 @@
+                            <div data-bind="compose: {model: schedule, view: '/assets/html/reports/EditScheduleViewModel.html'}"></div>
+                        </div>
+                    </div>
+
+                    <!-- Report Recipients -->
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h3 class="panel-title">Recipients</h3>
+                        </div>
+                        <div class="panel-body">
+                            <div data-bind="foreach: recipients">
+                                <div>
+                                    <button type="button" class="close right" aria-label="Close"
+                                            data-bind="click: function() { $parent.removeRecipient($index); }">
+                                        <span class="glyphicon glyphicon-remove text-danger"></span>
+                                    </button>
+                                    <div class="form-group">
+                                        <label for="recipient" data-bind="text: label()"></label>
+                                        <input id="recipient"
+                                               class="form-control"
+                                               type="text"
+                                               data-bind="value: address, attr: {placeholder: placeholder()}">
+                                    </div>
+                                    <div class="form-group">
+                                        <label for="recipientFormat">Report Format</label>
+                                        <span class="glyphicon glyphicon-question-sign"
+                                              data-bind="popover: helpMessages['format']"
+                                              data-trigger="hover"></span>
+                                        <select id="recipientFormat"
+                                               class="form-control"
+                                               data-bind="options: availableFormats,
+                                                          optionsText: 'text',
+                                                          optionsValue: 'value',
+                                                          value: format">
+                                        </select>
+                                    </div>
+                                </div>
+                                <hr>
+                            </div>
+                            <div class="dropdown">
+                                <button class="btn btn-default dropdown-toggle"
+                                        type="button"
+                                        id="addRecipientDropdown"
+                                        data-toggle="dropdown"
+                                        aria-haspopup="true" aria-expanded="true">
```

Comment:
What is handling the toggle here? Does it make sure to re-set aria-expanded to false when it isn't shown?

Reply:
Bootstrap handles this when the dropdown is initialized and adds the appropriate event handlers. It immediately gets set to false on page load but I'll set it to that.

Although the fact that this works also indicates that bootstrap is being loaded into the global scope somehow.

Reply:
Nevermind, it's not being loaded globally here - it's loaded through the same module as above in`KnockoutBindings.ts`